app/build.gradle.kts

```
@@ -43,6 +43,8 @@ android {
 }
 
 dependencies {
+    implementation(project(mapOf("path" to ":domain")))
```

```suggestion
    implementation(project(":domain"))
```
위와 같이 사용할 수도 있어요. (이하 관련 내용 동일)